app/controllers/bookmarks_controller.rb

```diff
@@ -50,7 +45,8 @@ def update
       name: params[:name],
       reminder_at: params[:reminder_at],
       options: {
-        auto_delete_preference: params[:auto_delete_preference] || 0
+        auto_delete_preference: params[:auto_delete_preference] || 0,
```

Comment:
Is it worth using strong parameters here for both the create and update params? Sometimes its handy just to have a list of valid params that the controller endpoints might take.

Should we create a method here for the options hash, or maybe just the default auto_delete_preference value. Or should the default `0` be added somewhere else and not be in the controller?

Reply:
Oh yes I will add `params.permit` for both endpoints. And yeah I will move the default to the `BookmarkManager`, makes more sense there :+1: